README.md

```diff
@@ -31,10 +31,12 @@ Once the dependencies are installed, you can build AIKIDO using [CMake]:
 ```shell
 $ mkdir build
 $ cd build
+$ source /opt/ros/indigo/setup.bash
```

Comment:
The actual path would be `source /path/to/my/workspace/devel/setup.bash` (`devel` is added), right?